pkgs/applications/audio/pyradio/default.nix

```diff
@@ -11,6 +11,15 @@ python3Packages.buildPythonApplication rec {
     sha256 = "04asw5alkkf2q5iixswarj6ddb0y4a6ixm7cckl6204jiyxpv6kc";
   };
 
+  propagatedBuildInputs = with python3Packages; [
+    dnspython
+    requests
+  ];
+
+  prePatch = ''
+    sed -i 's/distro = None/distro = NixOS/' pyradio/config
```

Comment:
Note that not all usage of nixpkgs is on NixOS.
This could feel weird for someone using nix+nixpkgs on another Linux or on Darwin.

Maybe `distro = nixkgs` is more appropriate? Maybe @contrun has an opinion on this.

Reply:
I agree.

Reply:
It's also preferable to use `substituteInPlace` over `sed`.
[Here is one example](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/audio/abcde/default.nix#L27-L28) of its usage in another package.

Reply:
Good point(s). I don't have any strong opinion on this. I choose `sed`, as it is the one being used in the README. I choose NixOS, as just `nix` is confusing (means `nothing` in german/dutch languages). `nixpkgs` is a nice solution.